app/src/js/action/point_cloud.ts

```diff
@@ -6,8 +6,8 @@ import { Vector3D } from '../math/vector3d'
 import * as types from './types'
 
 export const MOUSE_CORRECTION_FACTOR = 60.0
-export const MOVE_AMOUNT = 0.3
-export const ZOOM_SPEED = 1.1
+export const MOVE_AMOUNT = 0.15
+export const ZOOM_SPEED = 1.05
```

Comment:
Can you put the constants into one setting enum?

Reply:
I put them into an enum